pandas/core/dtypes/dtypes.py

```diff
@@ -120,6 +120,15 @@ class CategoricalDtype(ExtensionDtype):
         Must be unique, and must not contain any nulls.
     ordered : bool, default False
 
+    Attributes
+    ----------
+    categories
+    ordered
+
+    Methods
+    -------
+    None
```

Comment:
Is the problem that if you don't include this 'None', that it then automatically generated the 'Methods' section and thus included too much?